arc/conformance/plugin/conformance.yaml

```diff
@@ -0,0 +1,15 @@
+sonobuoy-config:
+  driver: Job
+  plugin-name: akv-secrets-provider-arc-conformance
+  result-format: junit
+spec:
+  image: mcr.microsoft.com/oss/azure/secrets-store/provider-azure-arc-conformance:v1.0.1
+  imagePullPolicy: Always
```

Comment:
Any reason for setting image pull policy to `Always`? If not, we should set it to `IfNotPresent`